meson.build

```diff
@@ -0,0 +1,17 @@
+project('iso-chooser-menu', 'c')
+
+arch = run_command('dpkg-architecture', '-q', 'DEB_BUILD_ARCH', check:true)
+add_global_arguments(['-DARCH="@0@"'.format(arch.stdout().strip()),
+                      '-Wall', '-Werror', '-Wfatal-errors'],
```

Comment:
Wall is the default, because the builtin core option `warning_level=XXX` is `1`. You can override this as `meson setup -Dwarning_level=2` or via `project(..., default_options: ['warning_level=2'])`, but adding it yourself causes meson to print a warning:

```
meson.build:6: WARNING: Consider using the built-in warning_level option instead of using "-Wall".
```

Same goes for -Werror, except that is `'werror=XXX'`, defaulting to `false`. You can manually set it to true in default_options. Meson currently warns about it:
```
meson.build:6: WARNING: Consider using the built-in werror option instead of using "-Werror".
```

Reply:
@eli-schwartz - thanks so much for stopping by!  I have implemented your suggestions.